packages/cache/package.json

```diff
@@ -45,10 +45,13 @@
     "@azure/abort-controller": "^1.1.0",
     "@azure/ms-rest-js": "^2.6.0",
     "@azure/storage-blob": "^12.13.0",
-    "semver": "^6.3.1"
+    "@protobuf-ts/plugin": "^2.9.4",
```

Comment:
These have been moved from the top level `package.json`. And `@protobuf-ts/plugin` was bumped up to `2.9.4`. Not sure why were using (and still using) `@protobuf-ts/plugin": "^2.2.3-alpha.1` in artifacts